apps/frontend/src/components/global/groups/GroupModal.vue

```diff
@@ -166,9 +167,18 @@ export default class GroupModal extends Vue {
     const toAdd: ISlimUser[] = this.groupInfo.users.filter(user => !originalIds.includes(user.id));
     const toRemove: ISlimUser[] = this.group.users.filter(user => !changedIds.includes(user.id));
     const addedUserPromises = toAdd.map((user) => {
-      const addUserDto: IAddUserToGroup = {
-        userId: user.id,
-        groupRole: 'member'
+      let addUserDto: IAddUserToGroup
+      if(user.id === ServerModule.userInfo.id) {
+        addUserDto = {
+          userId: user.id,
+          groupRole: 'owner'
+        }
+      }
+      else {
+        addUserDto = {
+          userId: user.id,
+          groupRole: 'member'
+        }
```

Comment:
Since users are automatically added to their group, we should be able to hide them from the user select dropdown.

e.g in Users.vue, L133:

```js
if(!currentUserIds.includes(user.id)) {
```
->
```js
if(!currentUserIds.includes(user.id) && user.id !== ServerModule.userInfo.id) {
```

Reply:
I was thinking the same. Kind of redundant to add yourself to group if you just get added when the group is created